Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet.Droid/MainActivity.cs

```diff
@@ -25,4 +30,25 @@ protected override void OnCreate(Bundle savedInstanceState)
             LoadApplication(new App());
         }
     }
+
+    public class AndroidAnalyticsListener : Java.Lang.Object, IAnalyticsListener
+    {
+        public void OnSendingFailed(ILog log, Java.Lang.Exception e)
+        {
+            System.Console.WriteLine("OnSendingFailed*****");
+        }
+
+        public void OnSendingSucceeded(ILog log)
+        {
+            System.Console.WriteLine("OnSendingSucceeded*****");
+        }
+
+        public void OnBeforeSending(ILog log)
+        {
+            EventLog eventLog = log as EventLog;
```

Comment:
Why filtering here only and not other callbacks? Please remove type filter and maybe log type.